alertmanager/multitenant.go

```diff
@@ -41,11 +85,70 @@ var (
 		Help:      "Time spent requesting configs.",
 		Buckets:   prometheus.DefBuckets,
 	}, []string{"operation", "status_code"})
+	statusTemplate      *template.Template
+	allConnectionStates = []string{"established", "pending", "retrying", "failed", "connecting"}
 )
 
 func init() {
 	prometheus.MustRegister(configsRequestDuration)
 	prometheus.MustRegister(totalConfigs)
+	statusTemplate = template.Must(template.New("statusPage").Funcs(map[string]interface{}{
+		"printState": func(enabled bool) string {
```

Comment:
I would probably omit the `print` prefix everywhere, as just naming a function by the thing it returns is more common naming in Go. In Prometheus for example: https://github.com/prometheus/prometheus/blob/master/template/template.go#L119-L251